src/mdns.c

```diff
@@ -273,7 +273,7 @@ mdns_list_interfaces(multicast_if** pp_intfs, struct mdns_ip **pp_mdns_ips, size
                         // See https://docs.microsoft.com/en-us/windows/win32/winsock/ipproto-ip-socket-options
                         *intfs = htonl(current->IfIndex);
                 }
-                PIP_ADAPTER_UNICAST_ADDRESS_XP p_unicast = current->FirstUnicastAddress;
+                PIP_ADAPTER_UNICAST_ADDRESS_LH p_unicast = current->FirstUnicastAddress;
```

Comment:
Why?

Reply:
Good question, I was getting a warning that `current->FirstUnicastAddress` is a `PIP_ADAPTER_UNICAST_ADDRESS_LH` on my machine, but the correct fix is actually to use ` PIP_ADAPTER_UNICAST_ADDRESS`. Fixed